docker/bookkeeper/bookkeeperTeardown.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+HOST=`hostname -s`
+PRAVEGA_PATH=${PRAVEGA_PATH:-"pravega"}
+PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
+BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
+BK_CLUSTER_ROOT_PATH=/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}
+BK_zkServers=$(echo "${ZK_URL:-127.0.0.1:2181}" | sed -r 's/;/,/g')
+
+if [[ $HOST =~ (.*)-([0-9]+)$ ]]; then
```

Comment:
Please, add comments about what this script is doing and why it is needed.

Reply:
done